suite/tests/api/ir_x86.c

```diff
@@ -299,6 +299,88 @@ test_all_opcodes_3_avx512_vex(void *dc)
 #    undef INCLUDE_NAME
 }
 
+static void
+test_all_opcodes_mask_2_avx512_evex(void *dc)
```

Comment:
There are 3 args so why does the name have 2?

Reply:
I count it as mask + 2 = 3. That was more intuitive to me. Do you want to call it mask_3?

Reply:
The file ir_x86_mask_2args_avx512_evex.h should be 3 IMHO: there are 3 args.  That would then extend to here.

Reply:
If I see this, I would think it's a mask + 3 operands, both for the mask_3args and mask_3 name. I renamed nevertheless, done.